### PR TITLE
General Linux deps are wrong

### PR DESCRIPTION
**Pull request by gerdesj**
See https://github.com/home-assistant/home-assistant/issues/24095

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_docs/z-wave/installation.markdown [hunk 1]
```diff
@@ -157,15 +157,12 @@ On the Raspberry Pi you will need to enable the serial interface in the `raspi-c
 
 #### {% linkable_title Linux (except Hassbian) %}
 
-On Debian Linux platforms there two dependencies you will need to have installed ahead of time (included in `systemd-devel` on Fedora/RHEL systems):
```
Comment (fabaff) [inline source/_docs/z-wave/installation.markdown:160]: According to the [spec file](https://src.fedoraproject.org/rpms/systemd/blob/master/f/systemd.spec) is `libudev-dev` still handled by `systemd-devel`.

**Review by frenck — APPROVED** [review]
Thanks, @gerdesj! 👍

**Review by frenck — CHANGES_REQUESTED** [review]
Aah missed the RH part that fabaff points out...

**Comment by frenck** [comment]
A small and friendly notification @gerdesj 

Are you planning on moving forward on this PR? 
I didn't see any activity recently, while there are still things to do. I hope you consider picking it up soon! 👍

**Comment by gerdesj** [comment]
I misread @fabaff's comment and they are correct.  I over edited.  I deleted the phrase _"(included in `systemd-devel` on Fedora/RHEL systems)"_ in error.

I've poked at Github a bit but I'm not sure how to change my commit and resubmit.  Please either drop my commit or if you have a black belt in git please fix it - the rest is correct. I don't need attribution.

Sorry for being a pain.

**Pull request closed without merging**